app/views/works/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>Editing Dataset</h1>
 
-<%= render 'form', dataset: @dataset %>
```

Comment:
Oops...looks like I forgot to remove this in a previous PR when I ditched the "dataset" model and controller.